src/ray/raylet_client/raylet_client.h

```diff
@@ -25,6 +23,7 @@
 #include "ray/common/status.h"
 #include "ray/common/task/task_spec.h"
 #include "ray/rpc/node_manager/node_manager_client.h"
+#include "src/ray/protobuf/gcs.pb.h"
```

Comment:
why we have "src" prefix here (also in other places)

Reply:
See above.